spec/controllers/tree_controller_spec.rb

```diff
@@ -7,7 +7,7 @@
         get endpoint
 
         expect(response.status).to eq(200)
-        expect(response.content_type).to eq('application/json')
+        expect(response.media_type).to match("application/json")
```

Comment:
can you keep this as `eq` if we're going to use `media_type`?

Reply:
thanks. that slipped through the cracks